Source/Utilities/UI/UIColor+CydiaColors.h

```diff
@@ -22,5 +22,12 @@
 + (UIColor*) cydia_removingColor;
 + (UIColor*) cydia_tintColor;
 + (UIColor*) cydia_black;
++ (UIColor*) cydia_darkTableViewCell;
```

Comment:
Why not, instead of having isDarkModeEnabled flags scattered across a bunch of UI files, you simply use the existing color methods and return dark variants when dark mode is enabled?